<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributesTab</class>
 <widget class="QWidget" name="AttributesTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="edit_button">
       <property name="text">
        <string>Edit...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view_button">
       <property name="text">
        <string>View...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_optional_attrs_button">
       <property name="toolTip">
        <string>Loads optional attribute values</string>
       </property>
       <property name="text">
        <string>Load optional attributes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="filter_button">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
